android/build.gradle

```diff
@@ -0,0 +1,35 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    ext.kotlin_version = "1.3.31"
+    ext.rxAndroidVersion = "2.1.0"
```

Comment:
Looks like lines 5 - 10 are duplicates of `android/databases/build.gradle`.  You can delete this one